Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45065&attempt_id=d3bdabd4-4b60-4521-a1f7-a956bf867cc3)

Attempted: twemoji-color-font

<details><summary>Partial log (click to expand)</summary><p>

```
cp assets/twemoji-svg/1f50f.svg build/staging/1f50f.svg
inkscape -w 1000 -h 1000 -z -e /dev/shm/1f50f.png build/staging/1f50f.svg
2018-08-15 13:05:40.864 inkscape[28386:49043176] *** WARNING: Method userSpaceScaleFactor in class NSView is deprecated on 10.7 and later. It should not be used in new applications. Use convertRectToBacking: instead.

** (inkscape:28386): WARNING **: 13:05:41.141: File path "/dev/shm/1f50f.png" includes directory that doesn't exist.

make: *** [Makefile:107: build/svg-bw/1f50f.svg] Error 1
rm build/staging/1f50f.svg
builder for '/nix/store/rxs1ybx6f5mn1g6r6zqdcgrqjmlcgjfx-twemoji-color-font-1.4.drv' failed with exit code 2
error: build of '/nix/store/rxs1ybx6f5mn1g6r6zqdcgrqjmlcgjfx-twemoji-color-font-1.4.drv' failed
```
</p></details>

--- a/pkgs/data/fonts/twemoji-color-font/default.nix
+++ b/pkgs/data/fonts/twemoji-color-font/default.nix
@@ -38,6 +38,6 @@ stdenv.mkDerivation rec {
     downloadPage = "https://github.com/eosrei/twemoji-color-font/releases";
     license = with licenses; [ cc-by-40 mit ];
     maintainers = [ maintainers.fgaz ];
-    platforms = platforms.all;
+    platforms = platforms.linux;
   };
 }

